src/services/completions.ts

```diff
@@ -3309,10 +3309,9 @@ namespace ts.Completions {
     /** True if symbol is a type or a module containing at least one type. */
     function symbolCanBeReferencedAtTypeLocation(symbol: Symbol, checker: TypeChecker, seenModules = new Map<SymbolId, true>()): boolean {

```

Comment:
@andrewbranch I think you worked on this and related functions not long ago.  As far as I can tell it's all looking good, but if you want to take a look, this change impacts the caller functions `getTypeScriptMemberSymbols` and `shouldIncludeSymbol` (called by `getCompletionEntriesFromSymbols` to filter out symbols), in completions.